app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+#   configure do
+#    set :public_folder, 'public'
+#    set :views, 'app/views'
+#    enable :sessions
+#   #   # set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
+#   #   # set :session_secret, SecureRandom.hex(64)
+# end
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params.value?('')
+      redirect to '/tweets/new'
+    else
+      user = User.find(session[:user_id])
+      Tweet.create(user_id: user.id, content: params[:content])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.